### PR TITLE
Metaflow pathspec in Airflow UI

### PR DESCRIPTION
**Pull request by valayDave**
These changes add Metaflow Pathspec to the Airflow UI.

### CODE REVIEW

File: metaflow/plugins/airflow/airflow_utils.py [hunk 1]
```diff
@@ -401,9 +419,27 @@ class MetaflowKubernetesOperator(KubernetesPodOperator):
         This is later useful in the `Workflow.compile` where the operator's `expand` method is called and we are able to retrieve the xcom value.
         """
 
-        def __init__(self, *args, mapper_arr=None, **kwargs) -> None:
+        template_fields = KubernetesPodOperator.template_fields + (
+            "metaflow_pathspec",
+            "metaflow_run_id",
```
Comment (savingoyal) [inline metaflow/plugins/airflow/airflow_utils.py:424]: can we also set - `metaflow_task_id`, `metaflow_attempt_id`, `metaflow_step_name`, `metaflow_flow_name`?

**Review by savingoyal — CHANGES_REQUESTED** [review]

**Comment by romain-intel** [comment]
This needs to be rebased/merged on/with master for testing to pass (due to changes in bootstrap command args).

File: metaflow/plugins/airflow/airflow_utils.py (hunk 1 above)
Comment (valayDave) [reply metaflow/plugins/airflow/airflow_utils.py:424]: Done

**Comment by valayDave** [comment]
Ready to be merged. code is tested.

File: metaflow/plugins/airflow/airflow_utils.py [hunk 2]
```diff
@@ -394,16 +412,44 @@ class MetaflowKubernetesOperator(KubernetesPodOperator):
...
+            "metaflow_task_id",
+            "metaflow_attempt",
+            "metaflow_step",
```
Comment (savingoyal) [inline metaflow/plugins/airflow/airflow_utils.py:427]: minor nit - `metaflow_step_name`

**Review by savingoyal — DISMISSED** [review]
pending one minor fix, otherwise gtg!

**Comment by valayDave** [comment]
Mergeable.

**Comment by valayDave** [comment]
Will rebase after which it will be ready for merge.

**Comment by valayDave** [comment]
rebased. We can merge now

**Pull request merged**